guides/common/modules/proc_salt-importing-salt-states.adoc

```diff
@@ -0,0 +1,22 @@
+[id="salt_guide_importing_salt_states_{context}"]
+= Importing Salt States
+
+A Salt State configures parts of a host, for example a service or the installation of a package.
+You can import Salt States from your Salt Master to {Project}.
+The Salt Master configuration used above uses a Salt Environment called `base` which includes the Salt States stored in `/srv/salt/`.
```

Comment:
```suggestion
The Salt Master configuration in this guide uses a Salt Environment called `base` that includes the Salt States stored in `/srv/salt/`.
```

From IBM: 
above adj
Do not use to indicate a relative location in a document, as in “the above restrictions.” Use “previous” or
“preceding.”
Example: “The previous restrictions apply to...”

Reply:
Ok seriously I started looking up the IBM guide but its not in our README. Then I found its a book, like a real book on paper and Amazon (PDF). It is not free. Red Hat documentation team provides some extra information, but it looks to be complementary to the IBM guide: https://redhat-documentation.github.io/supplementary-style-guide/

Is my understanding correct that to efficiently contribute content, you need to buy the book?

Reply:
It's a well known professional standard that most enterprise level software technical writers follow, especially in large software companies. Myself and @maximiliankolb have discussed it before and I think he was getting a copy for ATIX. 
I would only ever review on this level for @maximiliankolb because it is his profession and also because we've talked about it. Any Red Hat writers should have a copy of this and to use it. 
I wouldn't hold an engineer to these standards or expect them to look it up. I would expect a tech writer reviewing to suggest or implement the changes. 
This is where the professional writer/editor quality standards would come in. This was part of the original agreement between upstream and downstream - that these standards would be upheld. 
Maximilian has always shown interest and also tried to comply to the best of his ability, so I take the time to explain. 

TL;DR - I do not expect a contributor to know this, but a tech writer reviewing, should.

Reply:
Applied.

I totally get what @lzap is getting at: contributing to free software should ideally not depend on resources not accessible to everyone. I got my copy and refer to it whenever I can.

I am thankful for your/everyone's review and comments and I try my best to address them all.